curriculum/challenges/english/03-front-end-libraries/redux/register-a-store-listener.md

```diff
@@ -32,7 +32,13 @@ assert(
 There should be a listener function subscribed to the store using `store.subscribe`.
 
 ```js
-(getUserInput) => assert(getUserInput('index').includes('store.subscribe('));
+(getUserInput) => assert(getUserInput('index').match(/store\s*\.\s*subscribe\(/gm));
+```
+
+The listener function `store.subscribe` should have a function 
```

Comment:
```suggestion
The listener function `store.subscribe` should have a function. 
```
Nitpicking with the dot.

 I'm also having troubles with wrapping my mind around what actually this sentence is saying. If calling `store.subscribe` a listener function is correct, it might be better to still avoid using it. To not confuse it with passed in listener function. It doesn't look great, but maybe something like below?
```suggestion
The `store.subscribe` should receive a function.
```